### PR TITLE
fix(plasma-website): Fix by design review

### PR DESCRIPTION
**Pull request by neretin-trike**
После дизайн ревью были исправлены следующие места:
1. Добавлен отступ снизу главного блока;
2. Уменьшенно расстояние между версией релиза и датой;
3. Исправлена страница с икокнами: раньше свойство для грида не применялось, поэтому вместо `as`  использовал `forwardedAs`

:baby_chick: Download canary assets:

[colors--canary.452.4545613906.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.452.4545613906.xml)  
[color_metro_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.452.4545613906.swift)  
[color_metro_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.452.4545613906.kt)  
[color_metro_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.452.4545613906.ts)  
[color_sberHealth_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.452.4545613906.swift)  
[color_sberHealth_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.452.4545613906.kt)  
[color_sberHealth_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.452.4545613906.ts)  
[color_sbermarket_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.452.4545613906.swift)  
[color_sbermarket_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.452.4545613906.kt)  
[color_sbermarket_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.452.4545613906.ts)  
[color_sberprime_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.452.4545613906.swift)  
[color_sberprime_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.452.4545613906.kt)  
[color_sberprime_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.452.4545613906.ts)  
[color_selgros_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.452.4545613906.swift)  
[color_selgros_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.452.4545613906.kt)  
[color_selgros_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.452.4545613906.ts)  
[color_smbusiness_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.452.4545613906.swift)  
[color_smbusiness_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.452.4545613906.kt)  
[color_smbusiness_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.452.4545613906.ts)  
[PlasmaTokensColor--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.452.4545613906.swift)  
[shadow_sbermarket_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.452.4545613906.ts)  
[typo_mage_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.452.4545613906.swift)  
[typo_mage_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.452.4545613906.kt)  
[typo_mage_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.452.4545613906.ts)  
[typo_plasma_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.452.4545613906.swift)  
[typo_plasma_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.452.4545613906.kt)  
[typo_plasma_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.452.4545613906.ts)  
[typo_ruler_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.452.4545613906.swift)  
[typo_ruler_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.452.4545613906.kt)  
[typo_ruler_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.452.4545613906.ts)  
[typo_sage_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.452.4545613906.swift)  
[typo_sage_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.452.4545613906.kt)  
[typo_sage_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.452.4545613906.ts)  
[typo_sbermarket_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.452.4545613906.swift)  
[typo_sbermarket_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.452.4545613906.kt)  
[typo_sbermarket_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.452.4545613906.ts)  
[typo_soulmate_ios-swift--canary.452.4545613906.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.452.4545613906.swift)  
[typo_soulmate_kotlin--canary.452.4545613906.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.452.4545613906.kt)  
[typo_soulmate_react-native--canary.452.4545613906.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.452.4545613906.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-452/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-452/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-452/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-452/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-452/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-452/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-452/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

File: website/plasma-website/components/roster/Main.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import { Container } from '@salutejs/plasma-b2c';
 
-export const Main = styled(Container).attrs(() => ({ as: 'main' }))`
+export const Main = styled(Container).attrs(() => ({ forwardedAs: 'main' }))`
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply website/plasma-website/components/roster/Main.tsx:4]: бомбанет, надо думать ка починить грид, я написал об этом в чате

**Pull request merged**